### ISSUE TITLE
Ruby test configs

### ISSUE THREAD

**Issue by DAddYE** [seedifferently/the-great-web-framework-shootout#17]
Check this: https://github.com/DAddYE/web-frameworks-benchmark

For ruby apps I suggest you to use `thin` or `mongrel` both are fast and doesn't use `fork` like Passenger do.

Really I've no experience with go/python/php but for best comparison you should check:
- Number of threads that the webserver can fireup
- Threaded or Forked for handle concurrency

Thank you!
Great job!

**Comment by seedifferently** [comment]
Thanks for your input. Could you be a little more specific about what you are suggesting?

I chose Passenger because I figured Apache would be a good baseline for everything (mod_php for the PHP tests, mod_wsgi for the Python tests, mod_passenger for the Ruby tests). Sure, the Go test doesn't fall into this category but I kind of consider that a different animal.

**Comment by DAddYE** [comment]
Yep, passenger is a awesome webserver/module, but (I think) other than mod_php and mod_wsgi to handle concurrency request like an `ab` test can do it fork the ruby process for each request.

This a great workaround to scale but isn't the best way to track performances, no?

You want to track how many concurrency request can handle 1 ruby process?

If so, there is a great feature in passenger.

See [here](http://www.modrails.com/documentation/Users%20guide%20Apache.html#_resource_control_and_optimization_options)

In particular I suggest some like:

``` apache
PassengerMaxPoolSize 1
PassengerMinInstances 1
PassengerMaxInstancesPerApp 1
```
